lib/app_funcs.sh

```diff
@@ -60,6 +60,17 @@ function compile_app() {
   cd - > /dev/null
 }
 
+function post_compile_hook() {
+  cd $build_path
+
+  if [ -f post_compile.sh ]; then
+    output_section "Executing post_compile.sh"
+    ./post_compile.sh || exit 1
```

Comment:
@ericmj Thank you for clarifying a lot for me :) I've been out of touch with this stuff for almost a year now.